2023-06-09 00:18:56,769 INFO    MainThread:90072 [wandb_setup.py:_flush():76] Current SDK version is 0.15.4
2023-06-09 00:18:56,769 INFO    MainThread:90072 [wandb_setup.py:_flush():76] Configure stats pid to 90072
2023-06-09 00:18:56,769 INFO    MainThread:90072 [wandb_setup.py:_flush():76] Loading settings from /Users/toph/.config/wandb/settings
2023-06-09 00:18:56,769 INFO    MainThread:90072 [wandb_setup.py:_flush():76] Loading settings from /Users/toph/Documents/Private/GitHub/MLOps-Zoomcamp/wandb/settings
2023-06-09 00:18:56,770 INFO    MainThread:90072 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-06-09 00:18:56,770 INFO    MainThread:90072 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-06-09 00:18:56,770 INFO    MainThread:90072 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'preprocess_data.py', 'program': '/Users/toph/Documents/Private/GitHub/MLOps-Zoomcamp/preprocess_data.py'}
2023-06-09 00:18:56,770 INFO    MainThread:90072 [wandb_init.py:_log_setup():507] Logging user logs to /Users/toph/Documents/Private/GitHub/MLOps-Zoomcamp/wandb/run-20230609_001856-di8w00zp/logs/debug.log
2023-06-09 00:18:56,770 INFO    MainThread:90072 [wandb_init.py:_log_setup():508] Logging internal logs to /Users/toph/Documents/Private/GitHub/MLOps-Zoomcamp/wandb/run-20230609_001856-di8w00zp/logs/debug-internal.log
2023-06-09 00:18:56,770 INFO    MainThread:90072 [wandb_init.py:init():547] calling init triggers
2023-06-09 00:18:56,770 INFO    MainThread:90072 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-06-09 00:18:56,770 INFO    MainThread:90072 [wandb_init.py:init():596] starting backend
2023-06-09 00:18:56,770 INFO    MainThread:90072 [wandb_init.py:init():600] setting up manager
2023-06-09 00:18:56,776 INFO    MainThread:90072 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-06-09 00:18:56,782 INFO    MainThread:90072 [wandb_init.py:init():606] backend started and connected
2023-06-09 00:18:56,786 INFO    MainThread:90072 [wandb_init.py:init():703] updated telemetry
2023-06-09 00:18:56,806 INFO    MainThread:90072 [wandb_init.py:init():736] communicating run to backend with 60.0 second timeout
2023-06-09 00:18:57,167 ERROR   MainThread:90072 [wandb_init.py:init():762] encountered error: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
2023-06-09 00:18:57,255 ERROR   MainThread:90072 [wandb_init.py:init():1170] It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
Traceback (most recent call last):
  File "/opt/homebrew/anaconda3/envs/mlops_zoomcamp/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 1152, in init
    run = wi.init()
          ^^^^^^^^^
  File "/opt/homebrew/anaconda3/envs/mlops_zoomcamp/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 768, in init
    raise error
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
2023-06-09 00:18:58,260 WARNING MsgRouterThr:90072 [router.py:message_loop():77] message_loop has been closed
